packages/patterns-react/.storybook/manager-head.html

```diff
@@ -6,7 +6,7 @@
 <meta property="og:site_name" content="IBM.com Library Patterns">
 <meta property="og:description" content="A collection of IBM.com patterns implemented using React and Carbon Design System.">
 <meta property="og:image" content="https://media.github.ibm.com/user/525/files/59e3bfde-b990-11e7-87ef-072e89a87719">
-<meta property="og:url" content="https://ibmdotcomlibrary-patterns.mybluemix.net">
+<meta property="og:url" content="https://ibmdotcom-patterns.netlify.com">
```

Comment:
@photodow should be `ibmdotcom-patterns-react.netlify.com`

Reply:
Thanks for catching!